icons/shield_mx_mx.svg

```diff
@@ -0,0 +1,5 @@
3-1.2303-11.776-1.8455-24.782-1.8455-39.019-.00002-4.0424.0879-8.4363.26364-13.182.17574-4.7454.43938-9.4909.79091-14.236.52727-4.9211 1.0545-9.5788 1.5818-13.973.52726-4.3938 1.1424-8.3484 1.8455-11.864 2.2848-13.533 6.2394-23.2 11.864-29 5.8-5.7999 13.973-8.6999 24.518-8.7001 10.018.00018 18.103 2.9881 24.255 8.9637 6.1515 5.8002 9.6667 14.149 10.546 25.046h-22.937c-.35158-3.6908-1.5819-6.4151-3.6909-8.1728-1.9334-1.9332-4.394-2.8999-7.3819-2.9-4.0425.00016-7.2062 1.4062-9.491 4.2182-2.2849 2.8123-4.1304 7.1184-5.5364 12.918-.87884 3.5153-1.5819 7.1184-2.1091 10.809-.35156 3.5153-.70308 7.2941-1.0546 11.336-.1758 3.8668-.35155 8.1729-.52728 12.918-.00003 4.7456-.00003 10.018 0 15.818-.00003 6.1517-.00003 11.6 0 16.346.17573 4.5698.35148 8.788.52728 12.655.35148 3.691.703 7.2941 1.0546 10.809.52724 3.3395 1.2303 6.8546 2.1091 10.546 1.406 5.8001 3.1636 10.282 5.2728 13.446 2.2848 2.9879 5.2727 4.4819 8.9637 4.4819 4.0424.00002 7.0303-1.6697 8.9637-5.0091 1.9333-3.3394 2.9-8.9637 2.9-16.873h22.937M774.51 416.65c-.0001 4.7456-.0001 10.37 0 16.873-.0001 6.3274-.35162 12.918-1.0546 19.773-.52737 6.8546-1.5819 13.621-3.1637 20.3-1.4062 6.5031-3.691 12.391-6.8546 17.664-2.988 5.097-6.9426 9.2274-11.864 12.391-4.7456 3.1637-10.721 4.7455-17.927 4.7455-7.3819 0-13.534-1.5818-18.455-4.7455-4.7455-3.1637-8.7001-7.294-11.864-12.391-2.9879-5.2728-5.2728-11.161-6.8546-17.664-1.4061-6.6788-2.3728-13.446-2.9-20.3-.52729-6.8546-.79093-13.446-.79092-19.773-.00001-6.3273-.00001-11.952 0-16.873-.00001-5.0969-.00001-10.809 0-17.137-.00001-6.503.26363-13.094.79092-19.773.70302-6.6787 1.8454-13.182 3.4273-19.509 1.5818-6.5029 3.8667-12.215 6.8546-17.137 2.9879-5.0968 6.8546-9.1393 11.6-12.127 4.9212-3.1635 10.985-4.7453 18.191-4.7455 7.2061.00018 13.182 1.582 17.927 4.7455 4.9212 2.9881 8.8758 7.0305 11.864 12.127 3.1636 5.0972 5.4484 10.897 6.8546 17.4 1.5817 6.3275 2.6363 12.831 3.1637 19.509.70294 6.679 1.0545 13.27 1.0546 19.773-.0001 6.3274-.0001 11.952 0 16.873m-23.464 0c-.00007-2.4605-.0879-5.8-.26364-10.018-.00007-4.2181-.17583-8.7879-.52727-13.709-.35159-5.0969-.87887-10.194-1.5818-15.291-.7031-5.2726-1.7577-10.018-3.1637-14.236-1.2304-4.2181-2.8122-7.6454-4.7455-10.282-1.9334-2.6362-4.3062-3.9544-7.1182-3.9546-3.3395.00016-6.1516 1.4062-8.4364 4.2182-2.1092 2.6365-3.8668 6.1517-5.2728 10.546-1.2304 4.2184-2.197 8.9639-2.9 14.236-.52731 5.2729-.96671 10.458-1.3182 15.555-.1758 4.9214-.26368 9.4911-.26364 13.709.17572 4.0426.2636 7.1184.26364 9.2274-.00004 1.9334-.0879 5.0092-.26364 9.2274-.00004 4.0425.0878 8.6123.26364 13.709.35148 5.0971.79088 10.37 1.3182 15.818.703 5.4486 1.6697 10.37 2.9 14.764 1.406 4.394 3.1636 8.085 5.2728 11.073 2.2848 2.8122 5.097 4.2182 8.4364 4.2182 3.3394.00002 6.0636-1.406 8.1728-4.2182 2.109-2.9879 3.7788-6.6788 5.0091-11.073 1.406-4.394 2.3727-9.3152 2.9-14.764.5272-5.4485.87872-10.721 1.0546-15.818.35145-5.097.43933-9.6667.26364-13.709-.00007-4.2181-.00007-7.2939 0-9.2274" transform="translate(-3.493 -6.358) scale(.02753)"/>
```

Comment:
Can we get away with omitting or abbreviating the “MEXICO” legend? It becomes a blur at this size at 1× resolution.

Reply:
Here is a version without the "MEXICO":

![image](https://user-images.githubusercontent.com/3254090/209038632-75e6dbf1-d4ad-4fc8-864c-0694fa564ce0.png)

Would it be better to shrink the crown perhaps?

Reply:
Version with shrunken crown:

![image](https://user-images.githubusercontent.com/3254090/209038952-f22339b8-8b1b-4886-b274-fd977aa97e05.png)

Reply:
Shrunken crown looks significantly better, and it's still recognizable.

Reply:
That looks much cleaner. If any other network in Mexico has a similar-looking shield with a different legend, we could add a banner on top as with the Texas networks.

Reply:
Shrunken heads it is.

![image](https://user-images.githubusercontent.com/3254090/209049202-c9448660-a766-4e40-85fc-f5e00a4881ae.png)

Also scooted the padding up a bit since the crown is smaller now.